bgpd/bgp_fsm.c

```diff
@@ -1995,6 +1995,9 @@ bgp_connect_success(struct peer_connection *connection)
 	/* Send an open message */
 	bgp_open_send(connection);
 
+	if (peer->bfd_config)
```

Comment:
Do we need the call to bgp_peer_bfd_update_source in bgp_establish() if this is done here?  In other words why is it needed here -vs- there.

Reply:
What about the case of bgp_connect_success_w_delayopen?  Won't that not work?

Reply:
repush it for `bgp_connect_success_w_delayopen`, which is also with this problem.

Reply:
update the commit log: the bgp_establish can’t be reached for bfd strict mode without this commit in this case.